src/qc_data_preparation/helpers/dataset.py

```diff
@@ -90,3 +95,143 @@ def _exists(self) -> bool:
 
     def _describe(self) -> Dict[str, Any]:
         return dict(filepath=self._filepath)
+
+
+class PTModelDataset(AbstractDataSet):
+    def __init__(
+        self,
+        filepath: str,
+        load_args: Dict[str, Any] = None,
+        save_args: Dict[str, Any] = None,
```

Comment:
A default of `None` does not automatically imply the type is allowed. Annotate it explicitly:
```suggestion
        load_args: "Dict[str, Any] | None" = None,
        save_args: "Dict[str, Any] | None" = None,
```